### ISSUE TITLE
Open Source Design & DesignOpen meta discussion

### ISSUE THREAD

**Issue by simonv3** [opensourcedesign/jobs#26]
I started gathering some thoughts here: https://www.pinterest.com/simonv3/open-source-design-job-board/

Let me know if you want to join the board and I'll add you, or if there's an OSS version of Pinterest that has a bookmarklet like interaction thingy.

**Comment by bnvk** [comment]
@simonv3 niceness. You could also save those screenshots to the [opensourcedesign/patterns](/opensourcedesign/patterns) repo so we can build the collection of UI patterns... if you're so inspired ;)

**Comment by simonv3** [comment]
That is a whole lot more difficult than using the pinterest pin plug-in :P

**Comment by jancborchardt** [comment]
Pinterest is just relentless with their overlays and log-in walls. ;)

**Comment by simonv3** [comment]
Very true, maybe we can look at the tool the design open folks made, and
adapt it to general gh repositories?

**Comment by simonv3** [comment]
This has been closed now that we have an identity!

**Comment by GarthDB** [comment]
Hey - we might want to rethink something here. We have two job boards doing some of the same stuff with the same logo.

**Comment by GarthDB** [comment]
Here's a screenshot with the branding. 

![image](https://cloud.githubusercontent.com/assets/125516/7359437/2b05d772-ecf3-11e4-8eb6-28f75ca3e8ee.png)

**Comment by bnvk** [comment]
@GarthDB I see your point. I have removed your icon as of commit 027581f6023cad7cd02b6df72dc85b4078cfd7f1

**Comment by jancborchardt** [comment]
@GarthDB I'd also suggest removing the logo from https://thenounproject.com/term/open-source/23911/
Especially since you most probablyor apparently) don't want it to be used under the terms of public domain.

**Comment by GarthDB** [comment]
I'm not against the use of it. I just thought we should talk about it. It doesn't benefit either project to confuse potential users. If someone found a project on one board and searching for it again ended up on the other, it might be confusing.

**Comment by danilapellicani** [comment]
Hey people,

maybe I'm not aware of previous discussion but I was wondering why we don't
merge the two groups, the more we are the better the community will be.

Or am I missing something?


On Tue, Apr 28, 2015 at 9:36 AM, Garth Braithwaite <notifications@github.com

**Comment by bnvk** [comment]
@danilapellicani thanks for chiming in- yah this [has been](https://github.com/DesignOpen/designopen.github.io/issues/212) a [longtime discussion](https://github.com/opensourcedesign/resources/issues/14). I would have loved to merge our Github orgs months ago but there was hesitance & resistance. We are committed to using IRC (and improving the UX of that), they prefer Slack. We're going with simple markdown files + Jeykll for our Job board and Events page, they're building a slick app. They have a lot more tools & content already created. We're just starting out. From day one, one of my big goals is to figure how to get more designers working full time (with paid salaries) on FOSS projects, this was a point of contention early on. There are many similarities between our groups, but it seems there are also differences. Personally, i'm worn out trying to smooth over these differences and get to the state of "flow" and just create stuff!

@GarthDB I don't know what more there is to talk about. We had a lengthy chat in your Slack. You never posted explicit "don't use the icon in this way" rules and you released it into public domain, so we proceeded with our plans. All of our planning and steps towards this goal have been public. Our jobs repo has been growing... we all agreed on a [Design Hack Day](https://github.com/opensourcedesign/events/blob/master/2015-04-25-designs-and-hacks.md) on Sat to finally get something up. If you had glanced at the IRC room or logs, you could have seen this coming and chimed in... I was aware of the development of your job board and it is coming along great. I'm excited to see it grow and how it will be used. I **did not** think the same icon (with inverted coloring) would be an issue, but upon reflection, I think you might be right- logos get embedded in peoples brains and have a strong cultural association as being a symbol that means "same" and "singular" entity. Given the similarities of our goals and groups- there is that probability and it does not make sense to confuse users as there are also significant differences between our groups.

We just want to be producing stuff without friction or having to justifying or explain things- our hack day was the start of this and for the first time in months we actually have a site, and we're happy about that :smile: let's all just keep doing what we're doing without friction :+1:

**Comment by GarthDB** [comment]
@bnvk I still haven't said explicitly that I won't let anyone use our logo. I did release it with a very open license intentionally and I'm not taking that back. I was merely joining the discussion about it and offering my two cents. I'm really not mad I saw it used here that way. We are creative people, if we want to use it in both places let's find a way to make it different enough that it helps our users. It'll still communicate open source, collaboration, and design but different enough that it would also communicate the differences between our boards.




That said, I don't think we have enough differences between our boards to keep them separate forever. We have almost met your job board requirements and until we do I'm happy to keep them separate. I don't want to be the friction that keeps you from getting anything done. Once we do have an app that meets all your needs as well, would you be against merging them?




Is the problem that our app is nodejs built and not a Jekyll app? We did not choose the tech based on what we wanted to work in, as much as it was chosen by the quality of the user experience for the end user and the future capable features of that platform. Node gives us many more options that a static blog generator. I do personally love Jekyll; we use it for our main blog and I've personally pushed Jekyll pretty far. I also love how quickly you've put something together and I don't think either of the groups have done anything wrong in the paths we've taken, but let's extend the paths to an intersection.




Or is the problem that I am inviting you to eventually shut down this board and move all your posts to ours? This isn't an ownership thing for me. We're not using any proprietary code, it is all open. If we were making different enough job boards I would invite you to help our students make our nodejs job board platform and invite you to use it as well.





I think we can have a single organization that uses slack and irc. We can have one org that helps get more paid foss design jobs and help designers do free as in beer design work in the open.




I hope you can keep working in the state of flow, but be open to this discussion of merging the organizations.



—

**Comment by una** [comment]
Hi All! Thank you so much for what you're doing :) I love the community surrounding open source, and I'm so happy that there is a push to expand that community and diversify it to include the skill set of designers. This is such an important conversation to have.

I've seen a lot of progression with the people I've talked to on how they think about the importance of design in open source. I really think the work we're all doing has a lot to do with this. My biggest concern is growing the open source community, and the best way to grow a technical community is to introduce it to non-technical people. The best way to find a skill set that doesn't exist within a group is to look outside of that group. 

That goal far surpasses this identity struggle. I think we need to take a step back and remember what we're all doing here. I don't want to introduce people to a fractured system,

What I'm most concerned with is where do I direct someone who wants to get involved in open source for the first time? Where do I direct a project maintainer if they are seeking design? If we want to hold two job board -- that's fine -- but what is the practical distinction? A lot of work has been done to both, and maybe we should have had this conversation earlier, but going forward, I'd love to talk about how to leverage both to be the most effective. Maybe we user to see which is more usable? I don't have the answer, but I'd like to at least get the answer to the first two questions here.

Having two orgs is fine too, maybe once they are more defined we can move forward and help each other more instead of still trying to define ourselves. I think creating distinctions may be important and leveraging those distinctions positively. 

In the end, I don't care which way we go, but I do care about helping and connecting people. We are an example for the designers and developers we're trying to reach out to.

**Comment by philshapiro** [comment]
Heartfelt thanks to all who got opensourcedesign off the ground. 

My name is Phil Shapiro and I'm an open source enthusiast at a public library in the Washington DC-area. I also serve as a volunteer community moderator for Opensource.com, a community resource sponsored by Red Hat. It's useful to know that Opensource.com currently gets about 600,000 page views per month -- and is heading towards a million page views per month. (Thanks for helping spread the word about it.) 


Opensource.com can be a useful tool for getting the word out about the vision for this initiative, as the appropriate time. The folks who manage this web site are very supportive of blog submissions, I've found. 


Here is one way I can help. If we have designers interested in interviewing each other on topics related to design and open source, I can mail out some Olympus digital audio recorders I own. With each person recording themselves on their own side during a Skype conversation (or other audio conferencing tool), the high quality audio files can then be merged using Audacity. I can volunteer time to do the merging and the removal of uhms and ahs. 


The result ends up something like this -- engaging audio interview with a static graphics. 


https://www.youtube.com/watch?v=bJgjdXbYd44 


There's nothing more powerful than the human voice to open minds and help bring new folks on board, imho. 


Phil Shapiro 




----- Original Message -----

**Comment by GarthDB** [comment]
It's a great idea @philshapiro. I already have one podcast and have been considering starting one specifically for open design.




There could be enough content for a network of shows though.



—

**Comment by simonv3** [comment]
I wrote a long post that was essentially a rant on how we're not getting anywhere with this conversation, but then I deleted it all, which is probably a good thing.

> What I'm most concerned with is where do I direct someone who wants to get involved in open source for the first time? Where do I direct a project maintainer if they are seeking design? If we want to hold two job board -- that's fine -- but what is the practical distinction?

Honestly? At the moment you should probably direct them to the one that has a listing of jobs. 

Personally I don't mind if DesignOpen takes over the jobs posted on this board, but I have two major concerns:

1. One of the reasons for using Jekyll is that it follows a lot of IndieWeb [best practices](https://indiewebcamp.com/principles). It's easy to get off the ground, the data is in human readable and understandable form, flexible, super easy to transport to other things, and doesn't tie our content down anywhere. Which leads to the question, who's developing your job board? Will it be easy to be picked up by designers (the likely stewards of this community) to continue tweaking it? Or will it need a Node.JS dev?

2. Who can post things to your job board? A good chunk of the jobs posted on `/jobs/` currently aren't hosted on Github, and it looks like only owners of github repositories can post their own repository. That doesn't make a lot of sense to me because the person concerned with design on an open source project might not be the owner (for example, me on some projects). I'm pretty concerned with this idea that open source = github that keeps on cropping up everywhere.

**Comment by bnvk** [comment]
@GarthDB @una I've got no problem joining our groups. What I *do* have a problem with (and I will really not spend more than the next 10 mins doing) is "discussing" joining our groups. As I see it, it's been a continuous "discussion" that has not lead to productive outcomes. This is *exactly* what I was hoping to avoid by swapping out your logo and moving on. We're all good people who want to make / create cool stuff, let's just *do* that :grin:  That said, if you want to join groups, here are some steps towards that goal that c(sh)ould have been two months ago:

**Unify to one Github org**
The amount of users in @opensourcedesign is significantly larger, so I think it only makes sense to move @DesignOpen repos here (even if the name is changed). @GarthDB @una you all already have full access to the org. You can do these steps right now. Nothing is stoping you. Meanwhile I (and perhaps others) do not have any sort of access to @DesignOpen and never have.

**Logo**
Swap out our normal FOSS logo with yours. I love that icon and would be happy to be represented by it. I think the others feel the same

**Starting using one Twitter account**
Again, the @opensrcdesign has more followers, so that seems like a clear choice (even if name changes) as it's just silly and time consuming to be retweeting each others things.

**Unify our websites into one site**
Looking at both right now, there are pretty clear differences in what exists and what is missing from both sites. This should not be that hard to imagine doing.

**Come back to IRC**
Start coming back and having discussions in IRC, again, there is larger group, it's more open, plus all you @DesignOpen folks are technical enough to do it. Heck, help me finish up this improved Shout theme and you'll have a more "Slack like IRC" that can be deployed.
- Agree on a name. Personally, I like our name better. I think people "get it" immediately upon hearing it, but hell, put this idea to vote. Both of our groups will have to figure out some processes of reaching consensus eventually, regardless of if we join or stay separate.
 
**Jobs Section**
Ours is currently live. Yours is not. I'm sure we're all competent enough to figure out how to morph the benefits of both approaches into one platform. But this is *only* a discussion I will have after the above steps are completed,

I realize, I can't speak on behalf of this whole community. Others can keep discussing if they like, but i'll just be tuning it out (unless I see actions in the list above) as I have a lot on my plate at the moment.

**Comment by mdik** [comment]
Sounds reasonable to me.

**Comment by jancborchardt** [comment]
@simonv3 totally agree.

@bnvk good proposal. I would only add that it seems perfectly fine to join the pages. Open Source Design has the Jobs part and a detailed About, DesignOpen has Articles and Resources.

**Comment by GarthDB** [comment]
@simonv3 The mvp is a nodejs app for github projects. The next version could extend to non github projects and even paid jobs; we could even import your jekyll md files. The rest of the stuff I've explained above. Sorry to take you to the level of ranting; not my goal.

@bnvk I know you don't want to discuss this more, and you're ready to take action, but I have a responsibility to everyone who's made a contribution to DO.

So, in summary, we close down:
* [DO's github org](github.com/DesignOpen) (maybe move a couple of the DO projects to OSD)
* [DO's twitter](http://twitter.com/designopen)
* [DO's website](http://designopen.org) (maybe move our articles)
* [DO's slack](http://osd.slack.com)
* DO's domain name
* DO's job board

And the new org uses the DO logo.

**Comment by jancborchardt** [comment]
> So, in summary, we close down:

Well, if we look at stuff in a negative light then we’re not going to achieve anything here. If we’re talking about merging two things, here and there will be compromises. But let’s see:

> DO's github org (maybe move a couple of the DO projects to OSD)

No, we move _all_ of em. Not maybe but for sure. We merge the orgs. Everyone from DesignOpen is already member of Open Source Design on Github.

> DO's twitter

Most definitely the DO Twitter design is better so let’s take that. We can also carry over retweets or whatnot in some way, but honestly Twitter is so ephemeral I don’t really see the problem.

> DO's website (maybe move our articles)

Not maybe, most definitely. The articles are damn awesome. And the resources as well.

> DO's slack

Well we can run that in parallel, but as Brennan correctly noted it’s proprietary and very closed down. We might figure out a way to bridge discussions?

> DO's domain name

I also have https://opensourcedesigners.org and it just redirects.

> DO's job board

I can’t really comment on that. I kind of agree with Simon that https://github.com/opensourcedesign/jobs has the content and is also very easily accessible to add content for non-developers. Also, there’s not the projects-only-from-Github-problem. However we should improve the approach of how to add projects, for example a combination of your connect-to-Github approach and a simple form for other stuff not on Github.

> And the new org uses the DO logo.

Yeah, as it’s a great logo.

**Comment by bnvk** [comment]
@GarthDB the action items you listed are what I'm talking about. However, you used the word "close" whereby I used the word "unify" which is how I see this gesture. I hope if you proceed, you see this as the later and not the former- social media accounts often change names and say "Now follow us at @newhandle"

But by all means, honor your relationships to the community that has supported bringing @DesignOpen to life thus. See how they feel about this. If it makes sense, proceed, if it doesn't let's let this sleeping dog lay :dog: :zzz: :wink:

**Comment by bnvk** [comment]
@GarthDB lastly, like @jancborchardt pointed out, I think we should *definitely* keep and merge in all your articles and resources and the resource collector as all of that is great :smile:

**Comment by GarthDB** [comment]
@jancborchardt I'm not trying to be negative, but realistic, and brief, or at least getting to the point faster.

* The DO github org doesn't have much to it other than the website (which DO'd be moving to OSD) the job board (which we'd be using OSD's), and the resource collector. We could move that, but the rest is duplicated effort.
* I don't really care about twitter much. It's just another way to help people find help. There are designers who aren't really in open source yet there, so maybe a good tool for recruitment.
* The articles we'd need to rework the osd site to move them over.
* The Slack I don't think anyone on OSD has desires to keep running.
* The job board: I'm not quite ready to abandon the nodejs work done. I think it has merit, and I would like to run with it a bit more. I think relying too heavily on github for an entry point to potentially new designers might make the barrier to entry too high. It is merely a hypothesis, but I'd like to test it.

@bnvk it's semantics. We would be making them inactive and redirecting. There is duplicate effort and projects, that's the point of the discussion of merging.

To clarify, we've stopped the discussion of merging the other way because OSD has a better name and more followers?

**Comment by GarthDB** [comment]
Let me spend some time discussing this with the DO team. Most of the team has day jobs that aren't related to DO or open source, we do discussions at weird hours, and probably less frequently than OSD.

If you have any more clarifications feel free to post them here, and I'll post any more questions while we review everything.

**Comment by jancborchardt** [comment]
And let’s not to forget that there’s other initiatives which are very similar:

* [Open Usability](http://openusability.org/) by @guiguru and [@janmuehlig](https://twitter.com/JanMuehlig). Basically the ancestor of all these initiatives. It was also a big resource and driving force for me personally. Unfortunately it lost steam over time and the initiators focus on other things. I talked and worked bunch with them and also added them to the @opensourcedesign organization here.
* [Open Karma](http://open-karma.com/) where @saltinejustine & @bastilian are involved. From the description on the site the plan sounded very much like a job board. But it seems stale since some time. Another reason to just join up together – invited to @opensourcedesign since some time already.
* [OpenDesign](http://opendesign.io/) by just @mkfnch I think. The website design is pretty good and I’d love to »close down« our current design in favor of adopting that for example. ;) However it’s not very open about progress and it’s just one person, so here also joining forces makes more sense – invited to @opensourcedesign as well.
* one-off posts or initiatives like: [Creatives Unite](http://suddengrey.tumblr.com/post/106806347411/creatives-unite), [DesignOpenSrc](http://designopensrc.com/) by @clintonhalpin, and all these little blog posts from people saying »open source needs better design«

It’s just a shame that most of these initiatives lead nowhere. Mostly due to being just »founded« by one person or a few people which could then not devote the time it takes to push an initiative like this. That’s why I (and Brennan, Simon etc as well) really think we need to unite.

And we need to leave behind our strong sense of ownership and wanting to start our own little project to be famous with. Because a bigger initiative and community will be more powerful. It also includes all designers working in open source projects already. GNOME, Mozilla, Ubuntu, Drupal, Wordpress, Wikimedia, TextSecure, etc etc. Also added everyone I know could think of and worked with to @opensourcedesign.

Because this is not a project – but a community consisting of all the people who work on open source design.

**Comment by skddc** [comment]
Didn't even know opendesign.io; looks nice indeed!

**Comment by GarthDB** [comment]
Ok, as I was walking to the office I thought of another idea. This isn't vetted by anyone else, just a raw idea, so a little patience would be nice.

Could we make [DO](http://designopen.org) the recruiting arm of [opensourcedesign](https://github.com/opensourcedesign).

Like @jancborchardt said this isn't a project it's an initiative. Does it matter if we have one github team or two? We can be on the contributing team of both (many are already) and collaborate efforts that way. Do we need to have the same website? If we are attracting two different audiences and funneling them to the same goal, who cares about the rest, as long as we can work together.

I think OSD is more appealing to those already associated with open source and I know DO tries to break down barriers for contributors who find the existing open source culture to be daunting or unapproachable.

What do you think of that idea?

**Comment by una** [comment]
A lot of the work I do now is changing the perspectives of designers who have a predisposition to be intimidated by the existing open source community. A few thoughts on the above topics:

1. **Job Board:**
We should have **one job board**. That's important to me for unity. It can also link to @clintonhalpin's work with the list of designers because I feel like there are integrations to be made there :) The node work that has been done in DesignOpen is honestly really nice -- the user experience of clicking a button instead of going through and submitting a pull request is easier and faster. There are also some great ideas there with Contribution docs. It can be continued to be developed while we adopt and direct everyone to a singular job board, which does make sense that it be the one that's currently available for use (the one on Open Source Design). When the DO one gets to a point of being ready enough, we can all review it and then port the projects over, keep the domain, and have the benefits of the improved experience. MVP, then enhancement.

2. **Identity:**
It's not necessarily a bad thing to have a few different organizations working on the same thing. Not a bad thing to have different handles and events. I think it's actually good. Take, for instance all of the work on women in tech right now. There are a variety of organizations all doing essentially the same thing, but the more the merrier in getting things done from a slightly different angle and appealing to a larger audience. NCWIT and Women Who Code and All Girl Hack Night are all in support of each other's work, have unique identities, and there is room for all of them. To me, **Open Source Design** <-- the title and all, makes sense as an international umbrella organization. I think we can all function under it but no single person (or small group of people) should be running it, just as no single person or group runs "Women in Tech". There should be no biases toward any of the other orgs within it. *That's why I really liked the idea of having a newsletter or podcast to support everyone (events, meetups, workshops, etc.) through it (that @philshapiro mentioned)* We can't all be in the same place at once, nor are we targeting the same audience per se. We can even encourage new organizations to grow as contributors to the  Open Source Design Community (maybe eventually we'll even have orgs like The Open Source Typographic Society -- TOSTS or the New York Open Source Designers, NYOSD). Check out the [SassMeetup](https://twitter.com/SassMeetup) account as an example.

*tl;dr let's all keep doing what we're doing -- all use the Open Source Design job board until the Design Open one is ready -- not let a singular group control Open Source Design -- keep individual identities -- promote others to start groups and spread the word -- do a newsletter or podcast or something to keep in touch with what everyone is working on and support each other*

Also, a sidenote on IRC -- I don't love using it as a chat client anymore since the emergence of Slack (which I use for work and other chat groups and planning committees). I don't think IRC should be a requirement -- this is also not very welcoming to most designers who I've tried to get to join the IRC channel (we can argue about it later though).

**Comment by GarthDB** [comment]
@una what if we had two job boards - one for jobs and one for projects looking for design contributions?

I think the mvp for DO's board is close to ready for the second one, and OSD's board is better suited for the first. Project board and job board.

**Comment by jancborchardt** [comment]
I do see @una's point that there are also other initiatives with multiple organizations. If we want to do that we could, but I still think it makes more sense to unify since it’s so similar and not geographically bound.

Open Source Design is definitely supposed to not be »owned« by specific people (neither by me nor Brennan nor anyone else) and is intended as umbrella platform mostly to push what already exists in open source design, and amplify it

> what if we had two job boards - one for jobs and one for projects looking for design contributions?

Honestly I think that is just an arbitrary distinction which will result in a very bad UX. Why would we honestly think about splitting that, were we not in this current situation?

**Comment by GarthDB** [comment]
@jancborchardt I see them being very different audiences.

I'm not currently looking for a paid job, but I do enjoy finding open source projects that need design help.

**Comment by GarthDB** [comment]
We can link to each other though (Something along conveys a message along the lines of, "looking for an open source design job?" or "looking for open source projects that need design contributions?")

**Comment by GarthDB** [comment]
I guess that's my recommendation - we point people looking to do professional open source design and people doing it as volunteer work to design open.

**Comment by gillisig** [comment]
@GarthDB does that really solve a problem to split it up like that? 

http://opensourcedesign.net/jobs/ already makes it pretty clear which jobs are paid and which are not. So it should be pretty easy for you to find open source projects that need design help.

**Comment by GarthDB** [comment]
@gillisig it's a different audience and really different things.

**Comment by GarthDB** [comment]
One has interviews and normal hiring stuff, maybe even relocating; the other is a request for volunteer help probably part time, probably temporary.

**Comment by simonv3** [comment]
@GarthDB I think I see what you're saying. The way that I see the DO project board working right now is making it easier for open source maintainers to put out a call for help. I think that's definitely a worthy cause and something that's missing right now. I think what would be extra cool is if I could sign up with my own github account to not necessarily post a project, but to see what projects I have starred have asked for help, maybe on a tag set defined by the job board. Have you thought of doing something similar (going beyond the wild discrepancy of github issue tags).

At the moment the "job board" on open source design will probably continue to get stuff posted on it, but I don't know if it'll get much more "work" done on it before we move to a server that isn't owned by github (when we can start doing automatic pull requests, like the DO resource collector).

On the topic of the merge: since previous conversations on this topic to me DO has kind of always been more of a formal agency and Open Source Design more of a lose community of people, both with the intent to get more people involved, both working on awesome projects with great goals. It's probably possible for the work of the two communities to cross pollinate and be involved in eachother's projects without merging. But, like @bnvk said, I don't think any of the open source design folks can make this decision for the Design Open folks.

**Comment by simonv3** [comment]
I'm going to close this issue because I think everyone is happy doing what they're doing? Feel free to re-open if you think otherwise.

**Comment by jancborchardt** [comment]
Yeah, let’s close this. http://opensourcedesign.net/jobs/ does not have any arbitrary restrictions, and the existing jobs on there are good examples. We just need to improve the design, but that’s a separate story.
